content/webapp/views/pages/stories/story/index.tsx

```diff
@@ -89,7 +89,9 @@ const ArticlePage: NextPage<Props> = ({ article, serverData, jsonLd }) => {
   }, []);
 
   useEffect(() => {
-    fetchLinkedWorks();
+    if (article.hasLinkedWorks) {
+      fetchLinkedWorks();
+    }
```

Comment:
Great shout! I was wondering it it'd be worth adding `!isInPicturesFormat` as a conditional, which we do for the display of the carousel, but it goes back to "but do we want the preview to work still". My assumption is yes, but something to confirm?